_i18n/en.yml

```diff
@@ -866,6 +867,37 @@ moneropedia:
     wallet: Wallet
     clsag: CLSAG
 
+tools:
+  intro1: Below is a list of third-party tools for interacting with the Monero ecosystem. <b>These tools are not vetted by the Getmonero team, see the disclaimer at the bottom of this page.</b> If a tool no longer supports Monero or you would like a Monero tool to be listed, please
+  intro2: open a GitHub issue and let us know
+  block-explorers: Block Explorers
+  xmrchain: xmrchain.net
+  moneroblocks: MoneroBlocks
+  exploremonero: Explore Monero
+  monerovision: MoneroVision
+
+  address-generators: Address Generators
+  moneroaddress: moneroaddress.org
+  xmrllcoins: xmr.llcoins.net
+
+  network: Network
+  monerohash-nodes: monerohsah.com - Monero node map
```

Comment:
Assign to this entry (`monerohash-nodes`) only the description (`Monero node map`) no need to have the entire string translated. The name of the website can be directly on index.md.Same for the other entries below.